User States.md

```diff
@@ -0,0 +1,19 @@
+# User States
+
+| Logged In Status (Local to App) | User Status (Server and App) | Can sync data | Can add data | How does the user get to this state                                                                                                                                                                                                                                                                     | Will access token be present? | Notification to show on Home Screen | Initial screen to show when opened | Should show lock screen |
+|---------------------------------|------------------------------|---------------|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|-------------------------------------|------------------------------------|-------------------------|
+| Not Logged In                   | Waiting For Approval         | No            | Yes                   | The user can be in this state in two ways:<br> 1. When they select a facility during registration and the user is stored locally before scheduling the background registration call<br> 2. When a user number match is found on the server during login, but the request OTP call has not yet been made | No                           | Waiting for approval                | Home                               | Yes                     |
```

Comment:
I was wondering if we should remove "The user will be in this state when…" prefix for every `How does the user get to this state`.  

> The user will be in this state when a phone number match has been found on the server during login, but the request OTP call has not been made yet

to

>A phone number match has been found on the server during login, but the request OTP call has not been made yet.

Rest everything else looks good to me.

Reply:
Done.